salt/states/elasticsearch.py

```diff
@@ -275,10 +275,16 @@ def index_template_present(name, definition, check_definition=False):
                     ret['comment'] = 'Cannot create index template {0}, {1}'.format(name, output)
         else:
             if check_definition:
-                definition_to_diff = {'aliases': {}, 'mappings': {}, 'settings': {}}
-                definition_to_diff.update(definition)
+                if type(definition) == str:
```

Comment:
I think using an `isinstance` check here is more clear.

Reply:
I'll take care of that